rides/__init__.py

```diff
@@ -1,17 +1,21 @@
 ####################################
 # File name: __init__.py           #
 # Author: Ayush Goel & Fred Rybin  #
+# Contributors: Joe Abbate         #
 ####################################
 from subprocess import check_output
 import datetime
 import os
+from flask_mail import Mail, Message
```

Comment:
Just curious, but you've checked for CVEs against this package, right?

Reply:
CVE is clear for flask_mail

Reply:
We also use `flask-mail` for several other projects. Did you have specific concerns @WillNilges?  (if so, slack dm?)

Reply:
I did not, I just saw a new package being added and wanted to make sure. (Probably shoulda checked our other projects)

🆗